src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -77,10 +77,16 @@ PROGRAM_INFO("AdaBoost",
     "predictions for a given test dataset.  A test dataset may be specified "
     "with the " + PRINT_PARAM_STRING("test") + " parameter.  The predicted "
     "classes for each point in the test dataset are output to the " +
-    PRINT_PARAM_STRING("output") + " output parameter.  The AdaBoost model "
-    "itself is output to the " + PRINT_PARAM_STRING("output_model") +
+    PRINT_PARAM_STRING("predictions") + " output parameter.  The AdaBoost "
+    "model itself is output to the " + PRINT_PARAM_STRING("output_model") +
     " output parameter."
     "\n\n"
+    "Note : The following parameter is deprecated and "
+    "will be removed in mlpack 4: " + PRINT_PARAM_STRING("output") +
```

Comment:
Perhaps it makes sense to be more specific here and use mlpack 4.0.0?.

Reply:
Yes